dissect/target/plugins/os/unix/modules.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+from typing import Iterator, Optional
+
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export, internal
+from dissect.target.target import Target
+
+ModuleRecord = TargetRecordDescriptor(
+    "linux/module",
+    [
+        ("string", "name"),
+        ("varint", "size"),
+        ("varint", "refcount"),
+        ("stringlist", "modules_referred"),
```

Comment:
```suggestion
        ("string[]", "modules_referred"),
```
Also, since it's originating from a directory called `holders`, does it not make more sense to rename this field to `holders`? Or `used_by`, to adhere close to the `lsmod` output?

Reply:
Agreed. Both name and type fixed.